community-operators/elasticsearch-operator/elasticsearch-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -11,6 +11,9 @@ metadata:
     certified: "false"
     description: |-
       The Elasticsearch Operator for OKD provides a means for configuring and managing an Elasticsearch cluster for tracing and cluster logging.
+      ## Prerequisites and Requirements
+      ### Elasticsearch Operator Namespace
+      The Elasticsearch Operator must be deployed to the global operator group namespace (e.g. **openshift-operators**)
```

Comment:
There's no need for this, since it will be deployed to this namespace by default.
Is there a reason it's required to be deployed in "openshift-operators" rather than another global operator group namespace?

Reply:
Isn't that the global namespace?  Are you suggesting to simply not explicitly identify it?

Reply:
There's no need to explicitly identify it because it's not specific to EO or CLO. Unless there's something in EO or CLO that requires them to be installed to a specifically named namespace, these aren't "requirements".

Reply:
@SamiSousa better?

Reply:
Yes, thank you! Could this change also be applied to the CLO description?

Reply:
CLO MUST be installed to 'openshift-logging' namespace.  It's the only way the metrics scraping works correctly.  It has to be in an 'openshift-' namespace.